xtask/src/semver_check.rs

```diff
@@ -48,17 +48,12 @@ pub fn minimum_update(
     let mut min_required_update = ReleaseType::Patch;
     for (_, report) in result.crate_reports() {
         if let Some(required_bump) = report.required_bump() {
-            min_required_update = match required_bump {
-                ReleaseType::Major
-                    if min_required_update == ReleaseType::Patch
-                        || min_required_update == ReleaseType::Minor =>
-                {
-                    ReleaseType::Major
-                }
-                ReleaseType::Minor if min_required_update == ReleaseType::Patch => {
-                    ReleaseType::Minor
-                }
-                _ => ReleaseType::Patch,
+            let required_is_stricter = (min_required_update == ReleaseType::Patch)
+                || (min_required_update == ReleaseType::Minor
+                    && required_bump == ReleaseType::Major)
```

Comment:
I know I told you to do this, but the middle branch now actually seems unnecessary. `required == Major` is already checked in the last part of the condition.

That way:
- if min == Patch, min can be set to required because required is always >= Patch.
- if min == Minor, only required = Major will overwrite it - but required == Minor can safely be ignored
- if required == Major, it will overwrite anything because everything is <= Major

At least I think this is the case

Reply:
indeed - wasn't too sure and tested with code - seems to work